### ISSUE TITLE
Formatter issues

### ISSUE THREAD

**Issue by msftrncs** [Serhioromano/vscode-st#29]
The formatter (capitalizer) has a couple issues, and exposes a grammar issue.

```iecst
TYPE Bob : (* test *) STRUCT
        A, B : INT; //test
        b : INT; //test
    END_STRUCT
END_TYPE

(* Bool Word Struct Constant 
```
![image](https://user-images.githubusercontent.com/26179051/106424376-ae8e5e00-6427-11eb-8a9a-566ccea1c7ee.png)

when formatted:
![image](https://user-images.githubusercontent.com/26179051/106424165-55262f00-6427-11eb-9ef6-2d94b9fc3c4c.png)

Note there are multiple issues. 

1. It capitalized a broken comment.  The comment doesn't end, and that lack of an end trips up the regex used to detect keywords, while ignoring comments and quoted strings.
2. It thought the start of the comment was a parameter list for a function call or operator `END_TYPE` so it moved it.  'keywords' and the 'end' block keywords should be excluded from this treatment.,
3. The TM grammar changed the scope of the keyword  `END_TYPE` when it has a parenthesis behind it.   Since  `END_TYPE` is a rigid keyword, it should appear very high in the processing in the grammar file, above anything that utilizes a more generic form.

 The `VAR_` keywords, but not `VAR` or `END_VAR`, seem to be missing from the capitalization behavior.  This is not shown above.

**Comment by Serhioromano** [comment]
That is strange. Here is the code 

```js
let regEx = new RegExp(`\\b(?:${functions.join('|')})\\b\\s+\\(`, "ig");
```

It should move only functions. 

The problem was in `function` array. Last element was empty.

**Comment by Serhioromano** [comment]
3. Looks like `VAR_*` is not considered at all. Also fixed that.

**Comment by Serhioromano** [comment]
New version released 1.9.1 with 2 fixes.
